Lib/http/server.py

```diff
@@ -639,6 +639,11 @@ class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):
 
     server_version = "SimpleHTTP/" + __version__
 
+    extensions_map = {
+        '': 'application/octet-stream', # Default,
+        **mimetypes.types_map
+    }
```

Comment:
This is missing the `.py`, `.c` and `.h` defaults as well. They certainly aren't guaranteed to be available.

Reply:
Oh, I didn't see that `types_map` is already partially initialized. This bit is fine then - leave it as it is.